### PR TITLE
add vittrack and result

### PR DESCRIPTION
**Pull request by lpylpy0514**
GSOC Realtime tracking model
opencv repo PR link is [here](https://github.com/opencv/opencv/pull/24201)

### CODE REVIEW

**Comment by zihaomu** [comment]
@lpylpy0514, please convert the excel to markdown.

File: models/object_tracking_vittrack/README.md [hunk 1]
```diff
@@ -0,0 +1,36 @@
+# Vit tracker
+
+Vit tracker(vision transformer tracker) is a much better model for  real-time object tracking. Vit tracker can achieve speeds exceeding  nanotrack by 20% in single-threaded mode with ARM chip, and the  advantage becomes even more pronounced in multi-threaded mode. In  addition, on the dataset, vit tracker demonstrates better performance  compared to nanotrack. Moreover, vit trackerprovides confidence values  during the tracking process, which can be used to determine if the  tracking is currently lost.
```
Comment (zihaomu) [inline models/object_tracking_vittrack/README.md:3]: extra spaces

File: models/object_tracking_vittrack/README.md [hunk 2]
```diff
@@ -0,0 +1,36 @@
+# Vit tracker
```
Comment (zihaomu) [inline models/object_tracking_vittrack/README.md:1]: Vit -> VIT ?

**Review by zihaomu — COMMENTED** [review]
Please update the description. Others look good

File: models/object_tracking_vittrack/vitTracker.onnx
[the remark this answers is not in the document]
Comment (fengyuentau) [reply models/object_tracking_vittrack/vitTracker.onnx:1]: Also simplify the model with `onnxsim`. Rename the model to `object_tracking_vittrack_2023sep.onnx`.

File: .gitattributes [hunk 3]
```diff
@@ -1,20 +1,15 @@
 # Caffe
 *.caffemodel filter=lfs diff=lfs merge=lfs -text
-
```
Comment (zihaomu) [inline .gitattributes:3]: unnecessary modifications

**Comment by asmorkalov** [comment]
@fengyuentau @zihaomu Please take a look on the PR. The main part in OpenCV is ready for integration and awaits for model integration.

File: .gitattributes [hunk 4]
```diff
@@ -19,4 +19,4 @@
 *.jpg filter=lfs diff=lfs merge=lfs -text
 *.gif filter=lfs diff=lfs merge=lfs -text
 *.png filter=lfs diff=lfs merge=lfs -text
-*.webp filter=lfs diff=lfs merge=lfs -text
+*.webp filter=lfs diff=lfs merge=lfs -text
```
Comment (zihaomu) [inline .gitattributes:22]: unnecessary modifications

File: models/object_tracking_vittrack/demo.py [hunk 5]
```diff
@@ -0,0 +1,78 @@
...
+            break
+        # Inference
+        tm.start()
+        isLocated, bbox = model.update(frame)
+        score = model.getTrackingScore()
+        tm.stop()
+        # Visualize
+        frame = visualize(frame, bbox, score, isLocated, fps=tm.getFPS())
+        cv.imshow('vittrack Demo', frame)
+        tm.reset()
```
Comment (zihaomu) [inline models/object_tracking_vittrack/demo.py:78]: An empty line at the end of the file

**Review by zihaomu — APPROVED** [review]
LGTM! 👍

File: models/object_tracking_vittrack/demo.py [hunk 6]
```diff
@@ -0,0 +1,79 @@
+import cv2 as cv
+import argparse
+
+# Check OpenCV version
+assert cv.__version__ >= "4.8.0", \
```
Comment (zihaomu) [inline models/object_tracking_vittrack/demo.py:5]: `>=` -> `>`

**Pull request merged**